src/common/transform_json.py

```diff
@@ -20,6 +22,71 @@ def search_evs_list(data, search_str):
         if search_str in item.keys():
             return item[search_str]
 
+def find_ncit_code(data, ncit_key):
+    values = []
+    
+    def search_dict(d):
```

Comment:
can you simplify this method by removing the inner function, search_dict and expose the logics under the parent function, find_ncit_code?